composer.json

```diff
@@ -18,13 +18,13 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.3",
-        "symfony/framework-bundle": "~2.3",
-        "symfony/doctrine-bridge": "~2.3",
+        "symfony/framework-bundle": "~2.3|~3.0",
+        "symfony/doctrine-bridge": "~2.3|~3.0",
         "phpcr/phpcr-implementation": "2.1.*",
         "phpcr/phpcr-utils": "~1.2.0"
     },
     "require-dev": {
-        "symfony/form": "~2.2",
+        "symfony/form": "~2.3|~3.0",
```

Comment:
We have a hard BC break in Symfony Form 3.0, with a forward compatibility layer added in 2.7 to be able to have code compatible with both 2.7 and 3.0. But form types in this bundle are using the 2.3+ way to configure options